docs/guide/snaps-rpc-api.md

```diff
@@ -596,3 +596,70 @@ await wallet.request({
   params: ['clear'],
 });
 ```
+
+### `snap_notify`
+
+::: warning Only Callable By
+
+- Snaps
+  :::
+
+#### Parameters
+
+```typescript
+interface SnapNotifyParams {
+  /**
+   * Enum type to determine notification type.
+   */
+  type: NotificationType;
+
+  /**
+   * A message to show on the notification.
+   */
+  message: string;
+}
+```
+
+- `Array`
+
+  0. `SnapNotifyParams` - An object containing the contents of the notification.
+
+#### Notification Type
+
+```typescript
+enum NotificationType {
+  /**
+   * A default browser notification
+   */
+  native = 'native',
+
+  /**
+   * A in-app notification stored in the metamask state.
+   * Can be found in the notification panel.
+   */
+  inApp = 'inApp',
+}
+```
+
+#### Returns
+
+`null` - This method doesn't return any data.
+
+#### Description
+
+Calling this method causes a notification to be added in the MetaMask UI.
```

Comment:
Only `inApp` notifications are shown in the UI. Can we distinguish here?

Reply:
"Calling this method shows a notification natively in the browser or in MetaMask" or something like that?